modules/core/include/opencv2/core.hpp

```diff
@@ -300,19 +300,19 @@ function does not copy src itself but simply constructs the border, for example:
 @endcode
 @note When the source image is a part (ROI) of a bigger image, the function will try to use the
 pixels outside of the ROI to form a border. To disable this feature and always do extrapolation, as
-if src was not a ROI, use borderType | BORDER_ISOLATED.
+if src was not a ROI, use borderType | cv::BORDER_ISOLATED.
```

Comment:
I'm against the overuse of `cv::`. In many OpenCV samples we write `using namespace cv;` to make the code more clean. And let's keep it clean